File: Lista/quest13/estaticos.py
class Matematica:
    @staticmethod
    def fatorial(n):
        if n < 0:
            return None
        if n == 0 or n == 1:
            return 1
        resultado = 1
        for i in range(2, n + 1):
            resultado *= i
        return resultado

if __name__ == "__main__":
    numero = 5
    print(f"Fatorial de {numero}: {Matematica.fatorial(numero)}")

File: Lista/quest13/estaticos.go
package main

import "fmt"

type Matematica struct{}

func Fatorial(n int) int {
    if n < 0 {
        return -1 // Indica que o fatorial não é definido para números negativos
    }
    resultado := 1
    for i := 2; i <= n; i++ {
        resultado *= i
    }
    return resultado
}

func main() {
    numero := 5
    fmt.Printf("Fatorial de %d: %d\n", numero, Fatorial(numero))
}
